content/article/20170915-imint-customers-and-products.md

```diff
@@ -0,0 +1,54 @@
+---
+date: 2017-09-15T22:00:00+02:00
+tags: [sales]
+title: Imint customers and products
+author: Mr IMINT
+---
+
+# Background
+
+Here's a list of known customers of Imint AB's Vidhance products, along with known products Vidhance is included in. Some of those products have not been communicated by Iming AB themselves. Some have though been communicated by the customer in question, for example on Twitter, and some have been uncovered through firmware analysis.
```

Comment:
Typo: `Iming AB`

Reply:
It would be great to link to the actual press releases and tweets.

Reply:
Yeah, I'll fix it